libraries/helpers.rb

```diff
@@ -0,0 +1,18 @@
+module HabitatCookbook
+  module Helpers
+    def hab(*command)
+      begin
+        if defined?(shell_out_with_timeout!)
```

Comment:
Replacing `a_to_s` with `clean_array` isn't working, it's sending the argument of the quoted string to the `hab` command. E.g.,

```
[10] pry(#<Chef::Provider::Package::Hart>)> shell_out_with_timeout!(*clean_array("hab", *command))
Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
---- Begin output of ["hab", "pkg path core/redis"] ----
STDOUT:
STDERR: error: The subcommand 'pkg path core/redis' wasn't recognized
	Did you mean 'pkg'?

If you believe you received this message in error, try re-running with 'hab -- pkg path core/redis'

USAGE:
    hab [SUBCOMMAND]

For more information try --help
---- End output of ["hab", "pkg path core/redis"] ----
```